tools/omero/omero_filter.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import csv
+import json
+
+import ezomero as ez
+
+
+def filter_ids_ezo(user, pws, host, port, filter, id, value1, value2=None, tsv_file="filter_list.tsv"):
+
+    # Transform the id input in a list of integer
+    id = id.split(',')
+    id = list(map(int, id))
+
+    # Function to write tabular file from the ezomero output
+    def write_ids_to_tsv(data, header):
+        with open(tsv_file, 'a+', newline='') as f:
+            f.seek(0)
+            is_empty = f.tell() == 0  # Check if file is empty
+            writer = csv.writer(f, delimiter='\t')
+            if is_empty:
+                writer.writerow([header])  # Write the header
+            for item in data:
+                writer.writerow([item])  # Write each ID
+
+    try:
+        with ez.connect(user, pws, "", host, port, secure=True) as conn:
+            if filter == "filename":
+                fn_ids = ez.filter_by_filename(conn, id, value1)
+                write_ids_to_tsv(fn_ids, [f"Images with filename {value1}"])
```

Comment:
I would suggest to either remove the header line or prefix the header line with `#`. I would prefer the former. If you want to have column names in the Galaxy interface I would suggest to set the `column_names` metadata of the tabular output.